istio-telemetry/tracing/templates/service.yaml

```diff
@@ -30,14 +29,15 @@ metadata:
     app: {{ .Values.tracing.provider }}
     release: {{ .Release.Name }}
 spec:
+  type: {{ .Values.tracing.service.type }}
   ports:
-    - name: http-query
-      port: 80
+    - name: {{ .Values.tracing.service.name }}
+      port: {{ .Values.service.externalPort }}
```

Comment:
s/.Values.service.externalPort/.Values.tracing.service.externalPort/

Reply:
not sure if this will have impact for current user, because we're changing default port from `80` to `9411`. otherwise good to me.

Reply:
It's ok, I have tested it.